test/csuite/schema_abort/main.c

```diff
@@ -598,6 +632,8 @@ thread_ckpt_run(void *arg)
             first_ckpt = false;
             testutil_assert_errno(fclose(fp) == 0);
         }
+
+        testutil_tiered_sleep(opts, session, 0, &flush_tier);
```

Comment:
The call to `testutil_tiered_sleep` should replace the call to sleep at the top of the loop (and use the `sleep_time` arg).

Reply:
@raviprakashgiri29, can we do this?  `testutil_tiered_sleep(opts, session, sleep_time, &flush_tier);` at the top of the loop instead of calling `sleep` directly?

Reply:
@ddanderson I tried removing the sleep at line 577 to replace with testutil_tiered_sleep(opts, session, sleep_time, &flush_tier); but with this change the flush_tier call was never invoked. I think because the variable which drives the tier flushing `flush_tier` was never set to true.

Reply:
I'll leave this open while you investigate what's going wrong.